pkg/analysis/passes/gomanifest/gomanifest.go

```diff
@@ -139,41 +158,61 @@ func normalizeFileName(fileName string) string {
 	return strings.Replace(fileName, "\\", "/", -1)
 }
 
-func verifyManifest(manifest map[string]string, goFiles []string, sourceCodeDir string) error {
+func verifyManifest(
+	manifest map[string]string,
+	goFiles []string,
+	sourceCodeDir string,
+) ([]ManifestIssue, error) {
+	manifestIssues := []ManifestIssue{}
+
 	for _, goFilePath := range goFiles {
 		goFileRelativePath, err := filepath.Rel(sourceCodeDir, goFilePath)
 		if err != nil {
-			return err
+			return nil, err
 		}
 		// calculate the linuxSha256sum of the go file
 		linuxSha256sum, windowsSha256Sum, err := hashFileContent(goFilePath)
 		if err != nil {
-			return err
+			return nil, err
 		}
 		// check if the sha256sum is in the manifest
 		manifestSha256sum, ok := manifest[goFileRelativePath]
 
 		if !ok {
-			return fmt.Errorf(
-				"could not find file %s with hash %s in manifest",
-				goFileRelativePath,
-				linuxSha256sum,
-			)
+			manifestIssues = append(manifestIssues, ManifestIssue{
```

Comment:
previously we were directly returning an error. now we return an array of possible issues.